app/Group.php

```diff
@@ -19,7 +19,9 @@ class Group extends Model implements AuditableContract {
 
     protected $casts = [
         'include_child_groups' => 'boolean',
-        'show_unit' => 'boolean'
+        'show_unit' => 'boolean',
+        'start_date' => 'datetime',
+        'end_date' => 'datetime'
```

Comment:
does deleted_at not need this cast too?

Reply:
I don't think so. [`runSoftDelete()`](https://github.com/laravel/framework/blob/9448e01699d43761e8bcf3d2ff53d0e3efb37432/src/Illuminate/Database/Eloquent/SoftDeletes.php#L98) invokes `freshTimestamp()` like the other timestamp columns (which in turn uses Carbon).